x-pack/plugins/secops/server/lib/hosts/types.ts

```diff
@@ -20,6 +20,7 @@ export interface HostHit extends Hit {
     host: HostEcsFields;
   };
   cursor?: string;
+  firstSeen?: string;
```

Comment:
I know the rest of these fields are optional but I wonder why have `firstSeen` as optional when later it is always referenced as in:

```ts
firstSeen: bucket.firstSeen!.value_as_string,
```

At the moment, the `?` isn't getting you any additional safety unless I'm overseeing something.

Reply:
That `firstSeen` usage is actually from the `HostBucket` interface (below). It is required though so I went ahead and removed the `!`.